ELK/filebeat.yml

```
@@ -0,0 +1,20 @@
+  json.overwrite_keys: true
+
+processors:
+  - add_fields:
+      target: ''
+      fields:
+        environment: production
+
+output.elasticsearch:
+  hosts: ["https://elasticsearch:9200"]
+  api_key: "${ELASTICSEARCH_API_KEY}"
+  ssl: 
```

yamllint

[error] trailing spaces

(trailing-spaces)